CommandsManual.md

```diff
@@ -217,6 +219,56 @@ Attributes accepted as `<BUILD_IDENTIFIER>` (in order of search):
 
 An auth file is a `.impt.auth` file. It stores authentication and other information necessary to execute *impt* commands.
 
+**Note** An Auth file may contain more than one account's auth information, and can contain github info added using [impt auth github](#auth-github).
+
+Each account's auth information contains the following:
+
+- `endpoint`: an impCentral API endpoint (the API base URL).
+- `profileType`: "impCentral" or "github"
+- `accessToken`: Account's access token.
+- `expiresAt`: Expiration time mof the access token.
+- `loginKey`: login key for the account.
+- `userName`
+- `accounts`: an array of all accounts that can be accessed by this account.
+- `isDefault`: Boolean flag to indicate whether this is the default account to be used with *impt* commands. It is only included in one account and can be changed using [impt auth select](#auth-select).
+
+The Auth file is of the form:
+
+```JSON
+{
+    "USER_ACCOUNT_ID1": {
+        "endpoint": "https://api.imp.eatonem.com/v5",
+        "profileType": "impCentral",
+        "accessToken": "ACCESS_TOKEN",
+        "expiresAt": "2019-08-08T15:26:25.510Z",
+        "loginKey": "LOGIN_KEY",
+        "userName": "USERNAME",
+        "accounts": [
+            "USER_ACCOUNT_ID1",
+            "SHARED_ACCOUNT_ID1"
+        ],
+        "isDefault": true
+    },
+    "USER_ACCOUNT_ID2": {
+        "endpoint": "https://api.electricimp.com/v5",
+        "profileType": "impCentral",
+        "accessToken": "ACCESS_TOKEN",
+        "expiresAt": "2019-08-08T15:26:11.370Z",
+        "loginKey": "LOGIN_KEY",
+        "userName": "USERNAME",
+        "accounts": [
+            "USER_ACCOUNT_ID1",
```

Comment:
Is this supposed to be `USER_ACCOUNT_ID2?`

Reply:
And does the first account in the `accounts` list actually have to include the same ID as the key name for the user entry?

Reply:
It's `USER_ACCOUNT_ID1`, this array has all the accounts that can be accessed by the account ID that is the key name for the user entry as returned by the command `impt account list`